test/specs/scenarios/face.js

```diff
@@ -222,6 +222,24 @@ export const faceScenarios = (lang) => {
       cameraPermissions.verifyUIElementsOnTheCameraPermissionsScreen(copy)
     })
 
+    it('should be taken to the cross-device flow if browser does not have MediaRecorder API, liveness variant requested and photoCaptureFallback is disabled', async () => {
+      goToPassportUploadScreen(
+        driver,
+        welcome,
+        documentSelector,
+        `?language=${lang}&liveness=true&photoCaptureFallback=false`
+      )
+      driver.executeScript('window.MediaRecorder = undefined')
+      documentUpload.clickUploadButton()
+      uploadFileAndClickConfirmButton(
+        passportUploadImageGuide,
+        confirm,
+        'passport.jpg'
+      )
+      await sleep(2500)
```

Comment:
I think in general we want to avoid using `sleep` especially if it's more than 1s or so. We had some issues in the past with flaky tests when using this, if I recall correctly. Have you tried using something like `driver.wait(until.elementIsVisible(crossDeviceIntro.title()), 2500)`?

Reply:
this works perfectly and is much better! thanks :)